src/CMakeLists.txt

```diff
@@ -62,12 +63,16 @@ if (MSVC)
 		LINK_FLAGS "/subsystem:windows /ENTRY:mainCRTStartup")
 elseif(APPLE)
 	# Help CPack build the bundle
+	# Copying the icon file into *.app/Contents/Resources/
+	set_source_files_properties(${ICON} PROPERTIES
+		MACOSX_PACKAGE_LOCATION "Resources")
+
+	# Setting up Info.plist contents
 	set_target_properties(${PROJECT_NAME} PROPERTIES
-		RESOURCE glfw.icns
```

Comment:
Turns out RESOURCE is for shared libs in _framework_ packages. We don't have shared libs (unless someone chooses BUILD_SHARED_LIBS), and we don't build a framework  - so you're right, `RESOURCE` and `RESOURCE DESTINATION` are not needed.